### PR TITLE
Repubbies Pubbystation

### PR DESCRIPTION
**Pull request by pubby**
Here's an update for my map just to keep it fresh.

### Some bugfixes:

- Fixed hallway airlocks that were previously named "Diner" (they're now unnamed).
- Fixed improper areas in gateway room and xenobiology.
- Fixed bad camera tags.
- Fixed APC in engineering that was too close to the singulo.
- Fixed monastery shuttle lights floating in space.
- Added full glass windows to the emergency shuttle so that shuttle rotation code bugs stop letting all the air leak out. 

### Some requested changes:
- There's now a transit tube to the monastery, in addition to the shuttle.
- The turbine engine was moved from science to atmospherics.
- Added a deep fryer to the kitchen.

### Some wacky experimental redesigns:
- Medbay and science are now public hallways. This is to make the hallway topology more interesting to wander, and also to make lowpop have more interaction. At least, that's the idea.
- Genetics now has a fucking monkey sanctuary.
- Aux base is now in maintenance and it's fucking huge.

### Other changes:
- Most departments now have blue signs labeling them.
- Colored lights added.
- The bar can now be locked down with the push of a button.
- Viro and Xenobiology have windows looking into maint so that you can spy on them.

### New station map

[It's full of screenshot artifacts.](https://s4.postimg.org/9dsm5s323/stitched-fs8.png)

### New sprites

image

:cl: Pubby
experiment: PubbyStation's layout is now more public.
add: PubbyStation's turbine engine is now in atmospherics.
add: PubbyStation's monastery can now be reached by transit tube.
tweak: PubbyStation's aux mining base is now in engineering maint.
/:cl:

### CODE REVIEW

**Comment by Repukan** [comment]
Spicy

**Comment by Supermichael777** [comment]
we already have arrow signs

**Comment by ShizCalev** [comment]
3munj3t
South wall of the arrivals lounge has the wrong area.

3munlvi
Engie storage area doesn't fill the whole area.

image
random maint area on some windows outside atmos

3munqyg
viro's area is incorrect

image
pipes switch types in the morgue (under some cables)

image
unconnected cable under the door in the transit room

image
same under the port solar door as well

image
prison door

image
cable leading to nowhere above toxins

Would recommend scooting the Northern air vent in the nuke room over a tile since there will be a layer popping issue with the terminal right above it

File: _maps/shuttles/emergency_pubby.dmm [hunk 1]
```diff
@@ -1,629 +1,77 @@
-//MAP CONVERTED BY dmm2tgm.py THIS HEADER COMMENT PREVENTS RECONVERSION, DO NOT REMOVE
```
Comment (ShizCalev) [inline _maps/shuttles/emergency_pubby.dmm]: mapmerge :(

**Comment by pubby** [comment]
@ShizCalev Thanks. That's really helpful.

And I'm incompetent as FUCK right now.

**Comment by pubby** [comment]
I force-pushed an update to replace the previous mess with a single commit.

**Comment by Tacolizard** [comment]
pubby has always been the best map, and this reaffirms my stance

**Comment by octareenroon91** [comment]
Can high-pop miners get to the aux base? I don't remember them getting maint access except in lowpop.

**Comment by pubby** [comment]
>Can high-pop miners get to the aux base?

I don't think high pop miners can enter maint but I don't consider it much of an issue.

>Cryo is public-access but sleepers are not?

Yep. It takes 8 years of medical school to know how to open the sleeper room airlock :)

(the medbay layout is kinda an experiment)

**Comment by GunHog** [comment]
Why is Cargo not public access as well?

**Comment by pubby** [comment]
My theory was that public access departments would only work if they doubled as public access hallways, as that would bring people into the department who would not normally go there. For example, players traveling from the brig to engineering can now cut through medbay, which hopefully leads to more player interaction.

Cargo on the other hand is one big dead-end, and so it can't really function as a hallway. The only reason people would ever go to public-access cargo is if they wanted to do a cargo tech's job for him, and I think that reduces interaction rather than adds it.

**Comment by PromethTK11** [comment]
Changing the singularity for a super matter when

**Comment by PraiseRatvar** [comment]
i prefer if pubby was still married to the singularity engine since it makes the map more unique

**Pull request merged**